### PR TITLE
python3Packages.torchinfo: init at 1.6.2

### PR DESCRIPTION
**Pull request by storvik**
###### Motivation for this change

Add torchinfo python package. 

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
The commit should be named ``python3Packages.torchinfo: init at 1.6.1``.

**Comment by storvik** [comment]
Thanks for the input @SuperSandro2000

I think it should be fixed now. Also had to skip one test that required network connection to download pretrained weights for a model.

**Comment by SuperSandro2000** [comment]
@oborg build python3Packages.torchinfo

File: pkgs/development/python-modules/torchinfo/default.nix [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  disabledTests = [
+    # Skip as it downloads pretrained weights (require network access)
+    "test_eval_order_doesnt_matter"
+  ];
+
+  pythonImportsCheck = [ "torchvision" ];
```
Comment (SomeoneSerge) [inline pkgs/development/python-modules/torchinfo/default.nix:35]: Sanity check, should I replace this with `"torchinfo"`? Came up in another PR

Comment (SomeoneSerge) [reply pkgs/development/python-modules/torchinfo/default.nix:35]: @SuperSandro2000 I had to edit `torchinfo` in https://github.com/NixOS/nixpkgs/pull/222273/files#diff-3528d9b3fd4c65314034db8dd9d529ff644b1c2037e28ecfadae217825729746R46 anyway, so I updated `pythonImportsCheck` in a separate commit

**Pull request merged**